about.html

```diff
@@ -0,0 +1,175 @@
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">about</a>
+          </li>
+          <li class="nav-item">
+            <a href="#program" class="nav-link">program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#speakers" class="nav-link">speakers</a>
+          </li>
+        </ul>
+        <a href="">
+          <button class="nav-btn btn">CC Compaign</button>
+        </a>
+      </nav>
+      <div class="hero-body container">
+        <h2 class="hero-med-title">"Hello! Rugby fans"</h2>
+        <h1 class="hero-main-title">
+          Welcome to the home of Cape Town Rugby 10's
+        </h1>
+        <p class="hero-subtitle">
+          A fun festival where people from all around the world come together to
+          support their teams.
+        </p>
+        <h3 class="venue">3 – 4 FEBRUARY 2023 | ENTRIES OPEN 1 APRIL 2022</h3>
+      </div>
+    </section>
+
+    <section class="about-section">
+      <h2 class="about-title">About Quench Cape Town 10s</h2>
+      <div class="paragraph-container">
+        <p class="about-subtitle">
+          Quench Cape Town 10s is South Africa’s biggest social sport and
+          lifestyle event! Winning the 2016 Discovery Sport Industry Award for
+          “Best Live Experience” Event in South Africa.
+        </p>
+        <p class="about-subtitle">
+          It is the brainchild of rugby legends Bob Skinstad and Robbie Fleck
+          and long-standing mate Ron Rutland. Founded in 2008, starting off as a
+          rugby only tournament the event has since evolved into a multi-sport
+          event, with the addition of Netball in 2013, Beach Volleyball in 2016,
+          Dodgeball in 2017, Hockey and Soccer in 2018 and Running and esports
+          in 2019.
+        </p>
+        <p class="about-subtitle">
+          The rugby component of 10s is billed as the biggest ten-a-side rugby
+          tournament in the world, designed for fun but still underpinned by
+          national and international rugby federation rules. The event has, over
+          the years, featured some of the world's greatest International rugby
+          legends including Jonah Lomu, Carlos Spencer and Christian Cullen, as
+          well as other global sporting stars such as Graeme Smith and Justin
+          Kemp.
+        </p>
+        <p class="about-subtitle">
+          2018 marked the 10 Year Reunion of the Quench Cape Town 10s! The event
+          hosted 100 Rugby teams, 80 Netball teams, 60 Beach Volleyball teams,
+          20 Dodgeball teams, 14 Soccer teams, 10 Hockey teams, from all over
+          South Africa and abroad, taking part.
+        </p>
+      </div>
+    </section>
+
+    <section class="founders">
+      <div class="founder-images">
+        <img src="./images/robs.png" alt="robs image" />
+        <img src="./images/bobs.png" alt="bob image" />
+      </div>
+    </section>
```

Comment:
- [ ] - [ ] Kindly ensure that your about-page founder's section design matches the design guidelines given. Please take note of the background, typography, color, padding/margins 👍🏽 

| Design Guideline  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/190247943-4b409f40-dc78-4d31-83b3-5b60f9bc4cfd.png)  | ![image](https://user-images.githubusercontent.com/63473584/190248076-ddd8a6a7-2423-43ae-bb8d-5afa5ae99c6b.png)  |
| ![image](https://user-images.githubusercontent.com/63473584/190248203-7e7a9230-261d-4b5d-8cdf-3691e39a0963.png)  |![image](https://user-images.githubusercontent.com/63473584/190248348-2e0410ad-0fda-41a6-93dc-eca95b37b8ce.png)  |